build/dependencies.props

```diff
@@ -230,5 +230,10 @@
     <XunitExtensibilityExecutionPackageVersion>2.3.1</XunitExtensibilityExecutionPackageVersion>
     <XunitPackageVersion>2.4.0</XunitPackageVersion>
     <XunitRunnerVisualStudioPackageVersion>2.4.0</XunitRunnerVisualStudioPackageVersion>
+
+    <!-- Dependencies listed only to include otherwise-transitive references in Microsoft.AspNetCore.App.nuspec. -->
+    <MicrosoftDotNetPlatformAbstractionsPackageVersion>2.1.0</MicrosoftDotNetPlatformAbstractionsPackageVersion>
```

Comment:
Are these all the latest?

Reply:
No. The two below are out-of-date. Should I check Microsoft.Extensions.DependencyModel and all of the `System.*` packages above❔

Reply:
Yeah, it'd be better to have the latest for everything

Reply:
Agreed though I didn't check _everything_ in this file and instead focused on packages CoreFx, core-setup, extensions, and efcore